cli/package.json

```diff
@@ -15,21 +15,31 @@
         "src"
     ],
     "dependencies": {
+        "//": "****** Includes standard libraries from app-shell, so that tests work ******",
```

Comment:
This is needed for the same reason as we use `dependencies` in `cli-style`, right? To get them installed when a package installs this as a dep?

Reply:
Yeah - they’re not needed for production or development builds which get injected into the she’ll, but when running local tests they’re run outside the shell so they need to be able to find these peerdeps somewhere somewhere